src/libraries/System.Threading/tests/ThreadLocalTests.cs

```diff
@@ -316,7 +316,6 @@ private static void RunThreadLocalTest8Helper(ManualResetEventSlim mres)
         }
 
         [Fact]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.Mono, "This test requires precise stack scanning")]
```

Comment:
According to the comment, this test also looks like a candidate to be annotated with `[ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsPreciseGcSupported))]`.

Reply:
Yep